src/config/topics.js

```diff
@@ -438,6 +439,17 @@ export const railPlus = {
   menu: <RailplusMenu />,
   embedded: true,
   overlaySide: 'left',
+  messageConfig: {
+    eventType: railplusPdfDownloadType,
+    callback: (evt) => {
+      if (
+        evt.type === railplusPdfDownloadType ||
+        evt.data === railplusPdfDownloadType
+      ) {
+        document.getElementById(RAILPLUS_EXPORTBTN_ID).click();
```

Comment:
I'm wondering if the element could part of the callback parameter.

Reply:
No need I guess